.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
+# See https://help.github.com/articles/about-code-owners/
+
+/azure-pipelines.yml                           @dougbu
+/eng/*                                         @dougbu @NTaylorMullen
+/eng/common/                                   @dougbu
+/eng/scripts/                                  @dougbu @NTaylorMullen
+/src/Razor/                                    @ajaybhargavb @NTaylorMullen @rynowak
+/src/Razor/src/Microsoft.NET.Sdk.Razor/        @ajaybhargavb @NTaylorMullen @rynowak @pranavkm
```

Comment:
Not sure how this works but isn't it redundant to include people who are already subscribed to src/Razor?

Reply:
From https://help.github.com/en/articles/about-code-owners, it appears only one rule fires and later rules have precedence. So, the rules with @pranavkm in them override the /src/Razor/ default.